File: backend/src/main/java/com/rungo/api/domain/users/organizerApproval/repository/OrganizerApprovalRepository.java
package com.rungo.api.domain.users.organizerApproval.repository;

import com.rungo.api.domain.users.organizerApproval.entity.OrganizerApproval;
import org.springframework.data.jpa.repository.JpaRepository;

public interface OrganizerApprovalRepository extends JpaRepository<OrganizerApproval,Long> {
}

File: backend/src/main/java/com/rungo/api/domain/users/organizerApproval/repository/OrganizerApprovalRepository.kt
package com.rungo.api.domain.users.organizerApproval.repository

import com.rungo.api.domain.users.organizerApproval.entity.OrganizerApproval
import org.springframework.data.jpa.repository.JpaRepository

interface OrganizerApprovalRepository : JpaRepository<OrganizerApproval?, Long?>
